notebooks/noj_book/smile_classification.clj

```diff
@@ -12,9 +12,17 @@
 
 
 
+;; ## Smile classification models reference
```

Comment:
More context is needed, as in the Sklearn chapter. 
We should explain what Smile is, mention this is Smile v2, etc.